src/Neo/Ledger/Blockchain.cs

```diff
@@ -282,8 +282,11 @@ private VerifyResult OnNewBlock(Block block)
                 }
 
                 int blocksPersisted = 0;
-                uint extraRelayingBlocks = system.Settings.MillisecondsPerBlock < ProtocolSettings.Default.MillisecondsPerBlock
-                    ? (ProtocolSettings.Default.MillisecondsPerBlock - system.Settings.MillisecondsPerBlock) / 1000
+                // Check if this system's block time is faster than the default
+                TimeSpan blockGenTime = system.GetBlockGenTime();
```

Comment:
Why this? I do not thing it is necessary and quite strange.

Reply:
See https://github.com/neo-project/neo/pull/3622#discussion_r2020689634, this logic was originally ported from #3637.